lib/src/__tests__/DateTimePicker.test.tsx

```diff
@@ -83,7 +80,44 @@ describe('e2e - DateTimePicker', () => {
         .at(0)
         .text()
     ).toBe('2018');
-    // expect(component.find('ToolbarButton').at(1).text()).toBe('Jan 3');
+  });
+
+  it('prop: minDateTime – hours is disabled by date part', () => {
+    const { getByLabelText } = render(
+      <DesktopDateTimePicker
+        open
+        openTo="hours"
+        dateAdapter={new DateFnsUtils()}
```

Comment:
Haha smart

Reply:
Probably need to add some kind of `renderWithAdapter` or `renderWithContex`, so it will not be needed.

Reply:
Right, because we need to test the other date libraries?